morse: only define container_of if undefined

libubox.h also defines.

diff --git a/src/utils/morse.h b/src/utils/morse.h
index 17be9442d..19b216d83 100644
--- a/src/utils/morse.h
+++ b/src/utils/morse.h
@@ -99,9 +99,11 @@ enum morse_dot11ah_region {
  * Return the pointer to the start of a container when a pointer within
  * the container is known
  */
+#ifndef container_of
 #define container_of(ptr, type, member) ({\
 	const typeof(((type *)0)->member)*__mptr = (const typeof(((type *)0)->member) *)(ptr); \
 	(type *)((char *)__mptr - offsetof(type, member)); })
+#endif
 
 /**
  * morse_set_s1g_ht_chan_pairs - Set the s1g->ht channelisation pairs
-- 
2.34.1

